advanced/dapps/react-dapp-v2/src/helpers/tx.ts

```diff
@@ -41,3 +43,32 @@ export async function formatTestTransaction(account: string) {
 
   return tx;
 }
+
+
+export async function formatTestBatchCall(account: string) {
+  const [namespace, reference, address] = account.split(":");
+  // calldata for batch send
+  const receiverAddress = '0xc3cE257B5e2A2ad92747dd486B38d7b4B36Ac7C9'
```

Comment:
What is this address?

Reply:
Added my EOA address to test send ETH to it, can change it to something else.

Reply:
Yes lets change it to something else. Good practice is to add the address of Sepolia Faucet if we are sending Sepolia ETH